google/cloud/documentai_toolbox/wrappers/document.py

```diff
@@ -481,6 +481,35 @@ def from_gcs(
             gcs_input_uri=gcs_input_uri,
         )
 
+    @classmethod
+    def list_from_gcs(
+        cls: Type["Document"],
+        gcs_bucket_name: str,
+        gcs_prefix: str,
+    ) -> List["Document"]:
+        r"""Loads a list of Documents from Cloud Storage.
+
+        Args:
+            gcs_bucket_name (str):
+                Required. The gcs bucket.
+
+                Format: Given `gs://{bucket_name}/{optional_folder}/{operation_id}/` where `gcs_bucket_name={bucket_name}`.
+            gcs_prefix (str):
+                Required. The prefix to the location of the target folder.
+
+                Format: Given `gs://{bucket_name}/{optional_folder}/{target_folder}` where `gcs_prefix={optional_folder}/{target_folder}`.
```

Comment:
Can we use the same "format" example in the doc string for `gcs_bucket` and `gcs_prefix`?

Reply:
Done